libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -39,7 +39,7 @@ void NavEKF3_core::readRangeFinder(void)
             if (sensor == nullptr) {
                 continue;
             }
-            if ((sensor->orientation() == ROTATION_PITCH_270) && (sensor->status() == AP_DAL_RangeFinder::Status::Good)) {
+            if ((sensor->orientation() == ROTATION_PITCH_270) && (sensor->status() == AP_DAL_RangeFinder::Status::Good) && sensor->use_sensor()) {
```

Comment:
this really is a mess, a function that does this would make it a lot cleaner:
 - get_backend_orient_use(ROTATION_PITCH_270);
that would return the first in-range rangefinder with the given orient that has USE=1